GitLink.py

```diff
@@ -5,28 +5,22 @@
 import sublime_plugin
 import subprocess
 
-REMOTE_CONFIG = {
+HOSTINGS = {
     'github': {
-        'url': 'https://github.com/{0}/{1}/blob/{2}/{3}{4}',
-        'blame_url': 'https://github.com/{0}/{1}/blame/{2}/{3}{4}',
+        'url': 'https://github.com/{user}/{repo}/blob/{revision}/{remote_path}{filename}',
+        'blame_url': 'https://github.com/{user}/{repo}/blame/{revision}/{remote_path}{filename}',
         'line_param': '#L',
         'line_param_sep': '-L'
     },
     'bitbucket': {
-        'url': 'https://bitbucket.org/{0}/{1}/src/{2}/{3}{4}',
-        'blame_url': 'https://bitbucket.org/{0}/{1}/annotate/{2}/{3}{4}',
+        'url': 'https://bitbucket.org/{user}/{repo}/src/{revision}/{remote_path}{filename}',
+        'blame_url': 'https://bitbucket.org/{user}/{repo}/annotate/{revision}/{remote_path}{filename}',
         'line_param': '#cl-',
         'line_param_sep': ':'
     },
-    'codebasehq': {
```

Comment:
There are several people that rely on CodeBaseHQ still, so we'll need to leave that functionality in tact.

Reply:
Ah, I had overseen that! I added support for codebasehq (again), and tested it.